ch.sbb.scion.rcp.microfrontend.e3.app.demo/src/ch/sbb/scion/rcp/microfrontend/e3/app/demo/ApplicationStartup.java

```diff
@@ -57,7 +57,7 @@ public void postContextCreate(final MicrofrontendPlatform microfrontendPlatform,
             //
             ))))
         .applications(List.of(new ApplicationConfig().symbolicName("client-app").manifestUrl("http://localhost:4201/manifest.json"),
-            new ApplicationConfig().symbolicName("tms-kast-demo").manifestUrl("http://localhost:4200/assets/manifest.json"),

```

Comment:
Leave it for now...  but that reminds me on the next Issue: 
"i8n and externalizing strings"